src/lib/snark_worker/prod.ml

```diff
@@ -19,7 +23,7 @@ module Cache = struct
   let find (t : t) statement = Option.map ~f:snd (T.find t statement)
 end
 
-module Inputs = struct
+module Impl : Intf.Worker = struct
   module Ledger_proof = Ledger_proof.Prod
 
   module Worker_state = struct
```

Comment:
This does not seem to be a no-op as stated in the description. I'll check more tomorrow.

Reply:
How so?